elastic/kibana
# [vega] fix vega map validation errors crashing vega

Steps to test
* install sample web log data set
* Open vega visualization "[Logs] Visitors Map"
* Edit spec, add "maxZoom: 30" to `config.kibana` and click "Update" button

On Master, this causes an error that breaks vega.
<img width="500" alt="Screen Shot 2021-09-21 at 11 13 33 AM" src="https://user-images.githubusercontent.com/373691/134217427-d79f05ca-eff5-4cd8-8563-720efd357718.png">

After the fix is applied, the warning is displayed as expected
<img width="500" alt="Screen Shot 2021-09-21 at 11 16 34 AM" src="https://user-images.githubusercontent.com/373691/134217640-5ae1c8e5-f840-4a54-abc3-3fe8908db25b.png">
